src/fonts.css

```diff
@@ -1,30 +1,30 @@
 @font-face {
   font-family: 'SourceSansPro-Regular';
-  src: url('fonts/SourceSansPro-Regular.otf') format("opentype");
+  src: url('fonts/SourceSansPro-20190717-180900-5e1d162e.woff2') format('woff2');
```

Comment:
Maybe leave the otf file in there for IE11 users even though we aren't technically supporting them?

Reply:
Why though? It'll fallback. Also the app just won't work in IE soon

Reply:
Yeah fair, Edge still works.